one-container/pihole-unbound/Dockerfile

```diff
@@ -1,12 +1,14 @@
 ARG PIHOLE_VERSION
 FROM pihole/pihole:${PIHOLE_VERSION:-latest}
-RUN apt update && apt install -y unbound
+RUN apt update && apt install -Vy unbound wget php-cli php-sqlite3 php-intl php-curl
```

Comment:
would it possible to put support for this behind a flag? Just because some users may not want to use this utility (no idea why, seems cool!)